nsq/reader.py

```diff
@@ -726,14 +727,12 @@ def semver(v):
                 def cast(x):
                     try:
                         return int(x)
-                    except:
+                    except Exception:
                         return x
                 return [cast(x) for x in v.replace('-', '.').split('.')]
 
             if self.disabled.__code__ != Reader.disabled.__code__ and \
                semver(data['version']) >= semver('0.3'):
-                logging.warning('disabled() deprecated and incompatible with nsqd >= 0.3. ' +
-                                'It will be removed in a future release. Use set_max_in_flight(0) instead')
```

Comment:
Fixed the long line here by just removing this `logging.warning()`, while leaving the `warnings.warn()` just below. I figure pre-1.0 stuff will be removed pretty soon anyway.